### ISSUE TITLE
Figma board Clean-up

### ISSUE THREAD

**Issue by srhhnry** [cal-itp/benefits#1419]
Working through the following boards:

- [x] [Developer Benefits Full App](https://www.figma.com/file/SeSd3LaLd6WkbEYhmtKpO3/Benefits-(Full-Application)?type=design&node-id=5766%3A22626&t=kohHoOUTZCQB6bv8-1)
- [x] [Benefits Full App (desktop)](https://www.figma.com/file/SeSd3LaLd6WkbEYhmtKpO3/Benefits-(Full-Application)?type=design&node-id=5450%3A21241&t=kohHoOUTZCQB6bv8-1)
- [x] [Benefits Full App (mobile)](https://www.figma.com/file/SeSd3LaLd6WkbEYhmtKpO3/Benefits-(Full-Application)?type=design&node-id=9277%3A54013&t=kohHoOUTZCQB6bv8-1)

To ensure screens:

- follow a logical order
- elements within screens follow a logical order
- all elements and screens are titled and grouped correctly 
- elements are locked where appropriate

In other words, this is a clean-up so that the Benefits application design can be more easily understood as we bring in new people (e.g. product) or in case of the need for a temporary or permanent handoff (e.g. family leave).

**Comment by srhhnry** [comment]
Need to reconnect Desktop Prototype

**Comment by srhhnry** [comment]
All three Figma screens have been updated to include Google Sheets copy, properly grouped and named layers, and all elements within screens are named and organized correctly. There are small fixes pertaining to the Sheets sync and framing still to fix, but they are low-priority.
